lib/cisco_node_utils/interface.rb

```diff
@@ -271,6 +271,7 @@ def hsrp_bfd
     end
 
     def hsrp_bfd=(val)
+      return if val == hsrp_bfd
```

Comment:
why complicate code? It is just one check which is simple. The problem with interfaces is that there are so many types of logical interfaces and also some exist for only n7k (for ex. bridge). If new logical interfaces get created later, the code becomes unmanageable. Whether it is phyical or logical, this code just checks if the value is same as before and returns which should not add much time as this is just a boolean get.